### DISCUSSION TITLE
第十號交響曲大功告成

### DISCUSSION THREAD

**Discussion by henrywen547** [1101-GS3073-C/Python-Community-GS3073-C#50] [General]
貝多芬完成第九號交響曲後，於1827年過世，留下了第十號交響曲的幾個音符及一些作曲想法。
2019年，致力推廣音樂科技的奧地利組織卡拉揚研究所（Karajan Institute）負責人羅德（Matthias Röder）找來多位音樂家、AI專家，打算利用AI模仿貝多芬的風格，替他完成《第十號交響曲》
羅德:「之前沒有機器能創作這麼長的曲子，這非常獨特。」
事前準備:
整理貝多芬生前留下的《第十號交響曲》筆記，大致拼湊出貝多芬對《第十號交響曲》的想法
AI的學習目標:
  1. 從遺留作品與草稿學習貝多芬的創作脈絡
  2. 學習曾影響貝多芬的音樂前輩作品
  3. 熟悉基本曲式，並學習如何搭配樂器
作曲期中考
由記者、音樂學家、貝多芬專家分辨AI與原創的差異
結果:除了熟稔貝多芬流傳的《第十號交響曲》旋律專家，無人能分辨兩者差別
2021/10/9:譜成《第十號交響曲》並於德國波昂市（Bonn）首演
AI的學習歷程:從生疏到令人喜出望外
  團隊一位音樂專家甚至形容，看著AI作曲，「就好像看一位孜孜不倦的學生每天不斷練習、學習，表現也越來越出色」。
心得:
我找了《第九號交響曲》聽了幾個片段，其實我聽不出有甚麼極度相似之處，當然我也不是甚麼音樂專家，我其實對貝多芬也沒做太多研究。但我認為AI的這個應用感到有些震驚。我們之前可能提供了AI很多相同的數據來幫助我們判斷某些狀況，但這次，每個資訊似乎都獨一無二，一個音樂家的作曲風格會隨時間推演做出改變，會隨當下心境不同而不同。AI從具有自我判斷的能力，到具有創造的能力，不知道未來我們能再為AI開發什麼樣的能力?
資料來源:[https://technews.tw/2021/10/10/ai-completes-beethoven-symphony-no10/](url)

**Comment by awehao** [comment]
看來ai的成功，還是要建立在人類已經成功的基礎上

**Comment by Chenszuhong** [comment]
AI的用處可真大，感覺之後還有很多東西可以靠AI去完成

**Comment by larry40305** [comment]
可是，即使是這樣，AI作曲的出現仍然代表了一定的意義

**Comment by boson13579** [comment]
沒想到AI竟然能幫助我們聆聽到貝多芬未完成的第十號交響曲。

**Comment by 710388** [comment]
Ai 的創作 是否擁有專利？
